models/nxm/ethereum/nxm_ethereum_quotation_trades.sql

```diff
@@ -0,0 +1,79 @@
+{{ config(
+	alias ='trades',
+	partition_by = ['block_date'],
+	materialized = 'incremental',
+	file_format = 'delta',
+	incremental_strategy = 'merge',
+	unique_key = ['block_date', 'cover_block_number', 'statusNum', 'evt_tx_hash', 'evt_index'],
```

Comment:
Hi @sharkxff , the unique_key list here should match the columns in uniqueness test in schema.yml